CONTRIBUTING.md

```diff
@@ -1,3 +1,153 @@
 # Getting Involved
 
-If you have any experience with writing CLIs, Docker or Sails and Ember itself and you are interested in seeing this happening, just get in touch or open a new issue here.
\ No newline at end of file
+If you have any experience with writing CLIs, Docker, or with Sails and/or Ember
+themselves and you are interested in seeing this happening, just get in touch
+on our [Gitter chat](https://gitter.im/artificialio/sane) or open a new issue here.
+
+# Questions
+
+This is the repository and issue tracker for SANE. The community uses this
+site to collect and track bugs and discussions of new features. If you are
+having difficulties using SANE or have a question about usage
+please ask a question in the Gitter chat: https://gitter.im/artificialio/sane
+
+# Issues
+
+Think you've found a bug or have a new feature to suggest? Let us know!
+
+## Reporting a Bug
+
+1. Update to the most recent master release if possible. We may have already
+   fixed your bug.
+
+2. Search for similar issues. It's possible somebody has encountered this bug
+   already.
+
+3. Provide a demo that specifically shows the problem. This demo should be fully
+   operational with the exception of the bug you want to demonstrate. The more
+   pared down, the better. Issues with demos are prioritized.
```

Comment:
Sorry didn't see this before I pushed latest commit.  Will add some more details.  I'll also squash all these commits together tonight (don't think it can be done from the github GUI, can it?)